### ISSUE TITLE
Default classes + defining class from storyboard

### ISSUE THREAD

**Issue by fabiorodella** [erichoracek/Motif#79]
Great work on this library!

There are two features which would be particularly useful for me and that I'd like to tackle, but I just wanted to confirm if they are not already supported or have been considered and discarded:
- The ability to specify a default class for a specific `UIView` subclass (all `UIButton`s get the "DefaultButton" class by default, for instance).
- The ability to specify the class in the storyboard file through an `@IBInspectable` property.

The second one would require a way for the user to set a default global theme beforehand. 

Also, a big downside is that these would definitely involve swizzling, so it would be best to leave them in a subspec or something.

What do you think? If you like the idea I'll get started right away and submit a PR when done.

**Comment by fabiorodella** [comment]
This is what I came up with for setting classes from the storyboard in my app. It's pretty clean, works well and does not need any swizzling. I can submit a PR if this is of interest to anyone.

https://gist.github.com/fabiorodella/96ce303ac29e6eed3291

I gave up on setting default classes for now, as the swizzling needed to do it is way too aggressive to be in a library. Better to leave it up to the developers if they really need it.
